src/slurm_plugin/fleet_manager.py

```diff
@@ -319,7 +319,7 @@ def _launch_instances(self, launch_params):
                 logger.log(
                     log_level,
                     "Error in CreateFleet request (%s): %s - %s",
-                    err.get("ResponseMetadata").get("RequestId"),
+                    response.get("ResponseMetadata").get("RequestId"),
```

Comment:
can you put 
`response.get("ResponseMetadata", {}).get("RequestId"),` to avoid "None" errors

Reply:
done